kong/plugins/zipkin/reporter.lua

```diff
@@ -29,6 +29,11 @@ function zipkin_reporter_methods:report(span)
 	local span_context = span.context
 	local span_kind = span:get_tag "span.kind"
 	local port = span:get_tag "peer.port"
+	local zipkin_tags = {}
+	for k, v in span:each_tag() do
+		zipkin_tags[k] = tostring(v)
```

Comment:
Could you add a comment here about why the `tostring` is required?

Reply:
👍